Task3/js/components/ZombieElement.js

```diff
@@ -0,0 +1,34 @@
+function ZombieElement() {
```

Comment:
well...
you don't need an additional class just for the zombie's container-div
this whole logic should be implemented in scope of "Zombie" class